module-apps/apps-common/locks/handlers/SimLockHandler.cpp

```diff
@@ -348,8 +348,12 @@ namespace locks
         if (!simUnlockingNeeded_) {
             return sys::msgNotHandled();
         }
-
         simUnlockingNeeded_ = false;
```

Comment:
Won't this always set `simUnlockingNeeded_` to false? If yes, won't it disrupt anything?